modules/ivc_champva/app/services/ivc_champva/pdf_stamper.rb

```diff
@@ -9,15 +9,14 @@ class PdfStamper
     SUBMISSION_DATE_TITLE = 'Application Submitted:'
 
     def self.stamp_pdf(stamped_template_path, form, current_loa)
-      if File.exist? stamped_template_path
-        stamp_signature(stamped_template_path, form)
+      return unless File.exist?(stamped_template_path)
 
-        stamp_auth_text(stamped_template_path, current_loa)
-
-        stamp_submission_date(stamped_template_path, form.submission_date_stamps)
-      else
-        raise "stamped template file does not exist: #{stamped_template_path}"
-      end
+      stamp_signature(stamped_template_path, form)
+      stamp_auth_text(stamped_template_path, current_loa)
+      stamp_submission_date(stamped_template_path, form.submission_date_stamps)
+    rescue => e
+      Rails.logger.error "Error stamping PDF: #{e.message}"
```

Comment:
QQ: is there any chance that this message could contain PII?

Reply:
No PII. The most it will show are the file UUID, file number, and other general error information